### PR TITLE
10 reasons to give another chance to Firefox Nightly

### PR DESCRIPTION
**Pull request by Natim**
@glasserc @pascalchevrel ?

### CODE REVIEW

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+To be honest, starting to use Nightly today is a real huge step toward
+that goal.
+
+To use Firefox Nightly as a daily user is a simple and awesome way to
+help Firefox because it gives us feedback on the work we are doing
+ahead of time and also everyday. We don't have to wait for 18 weeks to
+get feedback on our work, we can have it the day after.
+
+By the time I write this article we have about 35000 daily users of Firefox
```
Comment (pascalchevrel) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: 38000

Comment (Natim) [reply content/2016.12.give.a.new.chance.to.nightly.rst]: 💃

**Comment by Natim** [comment]
If you want to read it more comfortably: http://www.servicedenuages.fr/en/drafts/give-firefox-nightly-a-new-chance.html

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+
+Conclusion
+----------
+
+I hope you are now convinced that you can help us to make a better
+Firefox by starting to use Firefox Nightly today.
+
+You can install the `desktop Firefox version <https://nightly.mozilla.org/>` as well as your
+`Android Firefox version <https://www.mozilla.org/fr/firefox/channel/android/>`
```
Comment (pascalchevrel) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: use https://www.mozilla.org/firefox/channel/android/ to have language negociation handled by mozilla.org

**Review by pascalchevrel — COMMENTED** [review]
Apart from a couple of nitpicks I mentionned in comments, that looks great to me, thanks!

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+
+Conclusion
+----------
+
+I hope you are now convinced that you can help us to make a better
+Firefox by starting to use Firefox Nightly today.
+
+You can install the `desktop Firefox version <https://nightly.mozilla.org/>` as well as your
+`Android Firefox version <https://www.mozilla.org/firefox/channel/android/>`
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: missing `_` for link format

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+Firefox is getting smaller and faster, new features are now tested
+through add-ons using `Test Pilot <https://testpilot.firefox.com/>`_
+instead of getting added to Firefox.
+
+This means less bugs and only great features by default.
+
+Also, this year 2017, Firefox is going to ship the
+`Quantum <https://wiki.mozilla.org/Quantum>`_ project which is a
+next-generation web engine. It benefit from some critical
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: nit: Web

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 5]
```diff
@@ -0,0 +1,212 @@
+10 reasons you should give another chance to Firefox Nightly
+############################################################
+
+:slug: give-firefox-nightly-a-new-chance
+:date: 2016-12-22
+:lang: en
+:url: give-firefox-nightly-a-new-chance
+:summary:
+    Help use make the best Mozilla Firefox of all the times by mid 2017!
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: Help use make?

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+:status: draft
+
+What is Nightly?
+================
+
+Firefox Nightly build also know as Nightly is a version of Firefox
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: also known

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+built from the Firefox source code everyday at night.
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: lots of *Firefox* in this sentence :)

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 8]
```diff
@@ -0,0 +1,212 @@
...
+===================================
+
+Anyone wants a stable Firefox and you don't have time for a crappy
+browser experience.
+
+Also, it was not always the case, Firefox Nightly is now much more
+stable than ever. The number of crashes has been schrinked down and it
+is getting lower really fast everyday.
+
+Also most of the bug that you might have will be fixed the day after.
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: most of the bugs?

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 9]
```diff
@@ -0,0 +1,212 @@
...
+Also, this year 2017, Firefox is going to ship the
+`Quantum <https://wiki.mozilla.org/Quantum>`_ project which is a
+next-generation web engine. It benefit from some critical
+parts of Gecko being rewritten in Rust which makes them 10 to 100
+times faster than before by using parallelization and GPU offloading.
+
+This is happening now and you will get it 3 months ahead.
+
+Also it is also a good opportunity to have a much recent and well
+tested code which means that you are less likely to have bugs in it.
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: How it is more *well tested*?

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 10]
```diff
@@ -0,0 +1,212 @@
...
+Then there are 4 versions of Firefox:
+
+- Firefox Nightly, which is a tag of Mozilla Central every day
+- Firefox Developer edition, which is a tag of Nightly every 6 weeks
+- Firefox Beta, which is a release of the previous Developer edition branch
+- Firefox Release, which is a release of the previous Beta branch
+
+With Nightly, the release is everyday, which means you will be the
+first one to benefit from the new feature, while other people will
+have to wait more than 3 months before being able to use it. (less
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `. (`: parenthesis after dot?

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 11]
```diff
@@ -0,0 +1,212 @@
...
+than 6 weeks in Nightly, 6 weeks for Developer edition, 6 weeks for
+Beta)
+
+Make a difference and help reduce the number of crashes for Mozilla Firefox users
+---------------------------------------------------------------------------------
+
+The more user we have in Nightly, the more stable other version of Firefox become.
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: *users*

*future versions of Firefox* ?

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 12]
```diff
@@ -0,0 +1,212 @@
...
+
+If everybody is waiting to use the last release of Firefox, it means
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: *is waiting* → *waits* ?

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 13]
```diff
@@ -0,0 +1,212 @@
...
+that Developer Edition and Beta step doesn't make any sense because
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: *steps don't*

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 14]
```diff
@@ -0,0 +1,212 @@
...
+
+By using Nightly, you will help us make a really great Firefox and as
+a benefit you will have this great Firefox experience 3 months ahead
+of schedule!
+
+Also most of the time, if you discover a problem with your Firefox
+Nightly, it will be fixed in the next release the day after, while
+with Firefox Release you will potentially have to wait for 6 weeks.
+
+Help add-ons developer to validate their add-ons is still working
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: *developers* 

*validate that* (?)

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 15]
```diff
@@ -0,0 +1,212 @@
...
+-----------------------------------------------------------------
+
+Firefox is transitionning from a complex and error prone way of
+writting add-ons to a standardized way of writting web extensions,
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: *writing*

Comment (leplatrem) [reply content/2016.12.give.a.new.chance.to.nightly.rst]: *Web extensions*

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 16]
```diff
@@ -0,0 +1,212 @@
...
+which are most likely to be compatible with Chrome and use well known
+web technologies which makes them simple to get started with.
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: Web

Comment (leplatrem) [reply content/2016.12.give.a.new.chance.to.nightly.rst]: which makes them easier to develop?

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 17]
```diff
@@ -0,0 +1,212 @@
...
+
+This is challenging for add-ons creators that already have a working
+add-ons that needs to be rewritten.
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: add-on

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 18]
```diff
@@ -0,0 +1,212 @@
...
+
+As a community we can help them. If they already have a Google Chrome
+add-ons it is most likely that they will be able to install it in
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: add-on, it is (singular + comma)

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 19]
```diff
@@ -0,0 +1,212 @@
...
+Firefox directly but Mozilla Firefox also expose more add-ons API than
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: directly. Plus, Firefox exposes more ...

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 20]
```diff
@@ -0,0 +1,212 @@
...
+Chrome and we will create more in 2017 to help as much as possible
+port previous add-ons to the web extension standard.
+
+As a user we can help to test this new add-ons to keep a good
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: these

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 21]
```diff
@@ -0,0 +1,212 @@
...
+
+To use Firefox Nightly as a daily user is a simple and awesome way to
+help Firefox because it gives us feedback on the work we are doing
+ahead of time and also everyday. We don't have to wait for 18 weeks to
+get feedback on our work, we can have it the day after.
+
+By the time I write this article we have only few handfuls of users
+for Firefox Nightly so your choice to use it makes a huge difference
+to us.
+
```
Comment (leplatrem) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: Also mention that you don't have to open bugs, the metrics and potential crash reports are already enough!

**Review by leplatrem — CHANGES_REQUESTED** [review]

**Comment by leplatrem** [comment]
Some illustrations or screenshots of new cool features like Containers would help make the long test easier to digest ;)
BTW, you don't mention that some features are only enabled in Nightly (ex. containers)

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 22]
```diff
@@ -0,0 +1,223 @@
+10 reasons you should give another chance to Firefox Nightly
+############################################################
+
+:slug: give-firefox-nightly-a-new-chance
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: I think `another-chance` sounds better here too.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 23]
```diff
@@ -0,0 +1,223 @@
+10 reasons you should give another chance to Firefox Nightly
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: Even though it might be grammatical, I don't like `to` here -- I prefer `give Firefox Nightly another chance`. If you are trying to structure the sentence to have `Firefox Nightly` at the end, maybe we can think of another formulation.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 24]
```diff
@@ -0,0 +1,223 @@
...
+:lang: en
+:url: give-firefox-nightly-a-new-chance
+:summary:
+    Help us make the best Mozilla Firefox of all the times by mid 2017!
+:status: draft
+
+What is Nightly?
+================
+
+Firefox Nightly build also known as Nightly is a version of Firefox
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `The Firefox Nightly build`, `Firefox: Nightly Build` (maybe?), or just `Firefox Nightly` (it seems like https://www.mozilla.org/en-US/firefox/channel/desktop/ just uses `Firefox Nightly`)

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 25]
```diff
@@ -0,0 +1,223 @@
...
+built from the source code everyday at night.
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `every day`

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 26]
```diff
@@ -0,0 +1,223 @@
...
+
+This means that Firefox updates itself everyday!
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: Same here. `everyday` is more like `ordinary`

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 27]
```diff
@@ -0,0 +1,223 @@
...
+
+
+Why aren't you using Nightly today?
+===================================
+
+Anyone wants a stable Firefox and you don't have time for a crappy
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `Everyone` wants

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 28]
```diff
@@ -0,0 +1,223 @@
...
+browser experience.
+
+Also, it was not always the case, Firefox Nightly is now much more
+stable than ever. The number of crashes has been schrinked down and it
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: I don't understand `it was not always the case`. Is this meant to refer to `more stable than ever`? Maybe rephrase to just `Although it wasn't always the case`. (`was not` sounds stiff.) `much more` is redundant with `than ever`, so pick one (I think `than ever`).

The past participle of `shrink` is `shrunk`. I don't like the phrasing `has been shrunk down` -- I think you could get rid of `down`, but how about `has been reduced`? Also, `getting lower` should be `smaller` if you're going to talk about shrinking a thing. I think `it is` feels a little stiff here, but if you want to keep it, `really fast` should be replaced with something more formal. Maybe replace the whole phrase with `and is being shrunk/further much further every day`?

Also, `every day`.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 29]
```diff
@@ -0,0 +1,223 @@
...
+is getting lower really fast everyday.
+
+Also most of the bugs that you might have will be fixed the day after.
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: I think you need more "connectors". How about "Besides," or "Additionally,"? (I'd replace "day after" with "the next day".)

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 30]
```diff
@@ -0,0 +1,223 @@
...
+You can always go in the `Help > About Firefox` menu to make sure you
+have the last update!
+
+
+Why you should actually give Firefox Nightly a try
+==================================================
+
+By installing Firefox Nightly you will:
+
+#. Have new Mozilla Firefox features 18 weeks before
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `earlier` instead of `before`, unless you're going to say before what, like `before they are released`

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 31]
```diff
@@ -0,0 +1,223 @@
...
+#. Have a much faster Firefox
+#. Make a difference
+#. Help reduce the number of crashes for Mozilla Firefox users
+#. Make sure Mozilla Firefox source code is in a workable state
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: I don't really understand how running Nightly has anything to do with the source code.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 32]
```diff
@@ -0,0 +1,223 @@
...
+#. Help add-ons developer to validate that their add-ons is still working
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `add-on developers`. `their add-ons are`

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 33]
```diff
@@ -0,0 +1,223 @@
...
+#. Have a better developer experience to develop your apps
+#. Give a really valuable contribution to the Mozilla project without coding
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: I think `Make a ... contribution`, not `Give`.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 34]
```diff
@@ -0,0 +1,223 @@
...
+#. Participate at building an Free Libre and Open Source browser that respect user privacy
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `a Free`. Comma after `Free`, and maybe another one after `Libre` depending on your tastes. `browser that respects`.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 35]
```diff
@@ -0,0 +1,223 @@
...
+#. Have a real impact on how the Web is built noawadays.
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `nowadays`

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 36]
```diff
@@ -0,0 +1,223 @@
...
+
+Have a much faster Firefox
+--------------------------
+
+Firefox is getting smaller new features are now tested through add-ons
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `is getting ... are now tested`? I think you can fix this by changing `are now tested` to `tested now`

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 37]
```diff
@@ -0,0 +1,223 @@
...
+using `Test Pilot <https://testpilot.firefox.com/>`_ instead of
+getting added to Firefox.
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: I don't really understand what you are trying to say by `getting added to Firefox` here. Instead of being shipped with Firefox? And then I don't really understand the flow of logic from here to talking about Quantum, which is a much bigger thing than can be shipped with Test Pilot.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 38]
```diff
@@ -0,0 +1,223 @@
...
+
+This means less bugs and only great features by default.
+
+Firefox is getting faster!
+
+This year 2017, Firefox is going to ship the `Quantum
+<https://wiki.mozilla.org/Quantum>`_ project which is a
+next-generation Web engine. It benefit from some critical parts of
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `benefits`

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 39]
```diff
@@ -0,0 +1,223 @@
...
+Gecko being rewritten in Rust which makes them 10 to 100 times faster
+than before by using parallelization and GPU offloading.
+
+This is happening now and you will get it 3 months ahead.
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: I prefer `early` here, but you could probably use something like `ahead of time` if you wanted to spice things up.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 40]
```diff
@@ -0,0 +1,223 @@
...
+
+Also Quantum is a lot more secure and the rewrite is a good
+opportunity to have a much recent and well tested code base which
+means that you are less likely to have bugs in it.
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `Not only that,`

`more recent and better-tested`, or `very recent and well-tested`. But I don't like `very recent`, I'd probably write `brand new`. But how can something that's brand new be well-tested? Seems contradictory.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 41]
```diff
@@ -0,0 +1,223 @@
...
+
+Have new Mozilla Firefox features 18 weeks before
+-------------------------------------------------
+
+Mozilla Firefox is on a fixed Release Calendar workflow called trains,
+which means that developper can plan their feature to land in a
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `developer`. `their features`?

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 42]
```diff
@@ -0,0 +1,223 @@
...
+certain version and coordinate other teams work (QA, OPS, Service
+developers, Deployments, Translations) for a certain date.
+
+In case the deadline is missed, they will have to wait for the next
+release 6 weeks later.
+
+Then there are 4 versions of Firefox:
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: I'd replace `Then` with something like `So`, but that's a little casual, so maybe `Today`

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 43]
```diff
@@ -0,0 +1,223 @@
...
+
+- Firefox Nightly, which is a tag of Mozilla Central every day
+- Firefox Developer edition, which is a tag of Nightly every 6 weeks
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: I think `Edition` is capitalized?

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 44]
```diff
@@ -0,0 +1,223 @@
...
+- Firefox Beta, which is a release of the previous Developer edition branch
+- Firefox Release, which is a release of the previous Beta branch
+
+With Nightly, the release is everyday, which means you will be the
+first one to benefit from the new feature, while other people will
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: Unless you're talking about a specific feature, `a new feature` (some generic hypothetical new feature) or `new features` (all of them). I prefer `new features`.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 45]
```diff
@@ -0,0 +1,223 @@
...
+have to wait more than 3 months before being able to use it (less
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: If you change to `new features`, then `use them`.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 46]
```diff
@@ -0,0 +1,223 @@
...
+than 6 weeks in Nightly, 6 weeks for Developer edition, 6 weeks for
+Beta).
+
+Not to say that some features are only activated in Nightly.
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `Not to say` here means something like `It isn't true that`, which seems a little strange. Maybe `Not only that, but`?

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 47]
```diff
@@ -0,0 +1,223 @@
...
+
+By using Nightly, you will help us make a really great Firefox and as
+a benefit you will have this great Firefox experience 3 months ahead
+of schedule!
+
+Also most of the time, if you discover a problem with your Firefox
+Nightly, it will be fixed in the next release the day after, while
+with Firefox Release you will potentially have to wait for 6 weeks.
+
+Help add-ons developers to validate that their add-ons is still working
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `add-on developers`. `add-ons are`

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 48]
```diff
@@ -0,0 +1,223 @@
...
+-----------------------------------------------------------------------
+
+Firefox is transitionning from a complex and error prone way of
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `transitioning`. `error-prone`

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 49]
```diff
@@ -0,0 +1,223 @@
...
+are most likely to be compatible with Chrome and use well known Web
+technologies which makes them easier to develop.
+
+This is challenging for add-ons creators that already have a working
+add-on that needs to be rewritten.
+
+As a community we can help them. If they already have a Google Chrome
+add-on, it is most likely that they will be able to install it in
+Firefox directly. Plus, Firefox exposes more add-ons API
+than Chrome and we will create more in 2017 to help as much as
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: I'm not sure if we actually have more APIs than Chrome..

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 50]
```diff
@@ -0,0 +1,223 @@
...
+possible port previous add-ons to the Web extensions standard.
+
+As a user we can help to test these new add-ons to keep a good
+experience for our users.
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: You use `user` twice in this paragraph to mean two different groups of people. Maybe change the first for something like `early adopter`.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 51]
```diff
@@ -0,0 +1,223 @@
...
+
+If you develop Web applications you might have had some problems to
+debug them with Firefox.
+
+We have a full team of developers dedicated to make Firefox the best
+tool for developping Web applications. This team is improving Web
+developer tools everyday and by using Nightly it will make your life
+easier.
+
+I wish that at some point you would even want to open Firefox because
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: Something about this phrase bothers me and I can't put my finger on what it is. I think it's because of the somewhat fixed phrase "I wish". Maybe `It's my wish` or `I hope`. Also, instead of `would` I'd probably put `you'll` (or `you will`, since you seem to avoid contractions).

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 52]
```diff
@@ -0,0 +1,223 @@
...
+
+Often people ask me how they can contribute to the Mozilla Firefox
+project.
+
+To be honest, starting to use Nightly today is a real huge step toward
+that goal.
+
+To use Firefox Nightly as a daily user is a simple and awesome way to
+help Firefox because it gives us feedback on the work we are doing
+ahead of time and also everyday. We don't have to wait for 18 weeks to
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: Instead of `everyday`, how about `immediately`?

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 53]
```diff
@@ -0,0 +1,223 @@
...
+get feedback on our work, we can have it the day after.
+
+By the time I write this article we have only few handfuls of users
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: Instead of `By the time I write this article`, the standard (slightly formal) way to express this is `As of this writing`, (although [some prefer](http://english.stackexchange.com/questions/199381/is-at-the-time-of-writing-correct#199392) `At the time of writing [this article]`).

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 54]
```diff
@@ -0,0 +1,223 @@
...
+Conclusion
+----------
+
+I hope you are now convinced that you can help us to make a better
+Firefox by starting to use Firefox Nightly today.
+
+You can install the `desktop Firefox version <https://nightly.mozilla.org/>`_ as well as your
+`Android Firefox version <https://www.mozilla.org/firefox/channel/android/>`_
+
+By the time you will be using Firefox Nightly, **if you see something, say something**.
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: `Once you are using`. I don't understand it but there's something weird about how conditionals work in English vs. French (it seems like a lot of the times, French uses the future when English uses the present). `By the time` would mean before it happens, but actually this is more like after it happens.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 55]
```diff
@@ -0,0 +1,223 @@
...
+
+You will have great chances to see it fixed the day after, while if
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: I'm pretty sure `chance` should be singular here (`a great chance`). Something bothers me about the word `great` here, I'd probably say a `good chance`, but what about something better than a good chance? Anyhow, it seems like [some native speakers](http://thehill.com/blogs/blog-briefing-room/news/302477-obama-dems-have-great-chance-of-regaining-house-majority-in-2014) use `a great chance` so I guess it's fine.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 56]
```diff
@@ -0,0 +1,223 @@
...
+you don't say anything it might never get fixed...
+
+`File a Bug in our bugtracker <https://bugzilla.mozilla.org/>`_ (you
+can now login with your Github account).
+
+Also in case of crashes, the bug report is sent automatically for you
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: I think you could use `However` instead of `Also`, especially if you add something like, `so you don't need to do anything` to the paragraph somewhere.

File: content/2016.12.give.a.new.chance.to.nightly.rst [hunk 57]
```diff
@@ -0,0 +1,223 @@
...
+with the detail that will hopefully help developpers to reproduce it
+as soon as you restart Firefox.
+
+
+Bonus : How to configure Firefox Nightly as my default browser in Linux?
```
Comment (glasserc) [inline content/2016.12.give.a.new.chance.to.nightly.rst]: No space before colon in English.

**Review by glasserc — COMMENTED** [review]
Wow, I can't believe I wrote 36 things. Sorry @Natim ... I know a second language is never easy.

File: content/2016.12.give.a.new.chance.to.nightly.rst (hunk 31 above)
Comment (Natim) [reply content/2016.12.give.a.new.chance.to.nightly.rst]: Nightly is a compiled version of the source code, by using Nightly you make sure the source code of Firefox is still a working recipe.

File: content/2016.12.give.a.new.chance.to.nightly.rst (hunk 37 above)
Comment (Natim) [reply content/2016.12.give.a.new.chance.to.nightly.rst]: Both things are not related but both makes Firefox faster.
